policyengine_us/parameters/gov/states/ca/chhs/adult/eligible.yaml

```diff
@@ -0,0 +1,11 @@
+description: California provides state-funded Medicaid to adults regardless of immigration status.
+values:
+  2023-12-31: false
+  2024-01-01: true
```

Comment:
Should the `false` date be earlier? Right we do not capture 2023 or any other year before that